forest/daemon/src/main.rs

```diff
@@ -49,14 +50,15 @@ fn ipc_shmem_conf() -> ShmemConf {
 }
 
 // Initiate an Event object in shared memory.
-fn create_ipc_lock() {
-    let mut shmem = ipc_shmem_conf().create().expect("create must succeed");
+fn create_ipc_lock() -> anyhow::Result<()> {
+    let mut shmem = ipc_shmem_conf().create().context("create must succeed")?;
     // The shared memory object will not be deleted when 'shmem' is dropped
     // because we're not the owner.
     shmem.set_owner(false);
     unsafe {
         Event::new(shmem.as_ptr(), true).expect("new must succeed");
```

Comment:
Can we handle errors here in a better way?

Reply:
Fixed